cli/cobra.go

```diff
@@ -159,15 +161,27 @@ func isFirstLevelCommand(cmd *cobra.Command) bool {
 	return cmd.Parent() == cmd.Root()
 }
 
-func commandVendor(cmd *cobra.Command) string {
-	width := 13
-	if v, ok := cmd.Annotations[pluginmanager.CommandAnnotationPluginVendor]; ok {
-		if len(v) > width-2 {
-			v = v[:width-3] + "…"
+func isPluginFirstLevelCommand(cmd *cobra.Command) bool {
+	return isPlugin(cmd) && isFirstLevelCommand(cmd)
+}
+
+func decoratedCommandName(cmd *cobra.Command) string {
```

Comment:
bike-shedding, but these are all commands, so perhaps just

```suggestion
func decoratedName(cmd *cobra.Command) string {
```

Reply:
Done